src/coreclr/inc/corinfo.h

```diff
@@ -1726,10 +1726,12 @@ struct CORINFO_FIELD_INFO
 
 struct CORINFO_THREAD_STATIC_BLOCKS_INFO
 {
-    CORINFO_CONST_LOOKUP tlsIndex;
-    uint32_t offsetOfThreadLocalStoragePointer;
-    uint32_t offsetOfMaxThreadStaticBlocks;
-    uint32_t offsetOfThreadStaticBlocks;
+    CORINFO_CONST_LOOKUP tlsIndex;              // windows specific
+    uint32_t offsetOfThreadLocalStoragePointer; // windows specific
+    size_t offsetOfMaxThreadStaticBlocks;
+    size_t offsetOfThreadStaticBlocks;
```

Comment:
```suggestion
    size_t offsetOfMaxThreadStaticBlocks;
    size_t offsetOfThreadStaticBlocks;
```
I think it made more sense for these two to be `uint32_t`.